website/docs/restricted-environment/index.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 2
+title: Restricted environment
```

Comment:
'environments' should be plural?

This section seems to be all about install in these environments. If so, shouldn't it move under the Installing section?

Reply:
I am preparing for the next move: #4270

Reply:
I would say the plural is feature-centric (we support multiple restricted environment situations), and the singular is user-centric (a user works in one environment). We want the docs to be user-centric.